### DISCUSSION TITLE
Issue while compiling ImageMagick with HEIF.

### DISCUSSION THREAD

**Discussion by harshitbhardwaj19** [ImageMagick/ImageMagick#6044] [General]
**ImageMagick version**
7.1.0-60 Q16-HDRI x86_64 20831

**Operating system**
centOS

**Operating system, version and so on**
CentOS Linux release 7.6.1810 (Core)

**Description**
Facing an issue while installing ImageMagick with support for heif. I have both libheif and libde265 installed. However, after running configure for ImageMagick (./configure --with-heic=yes), I am facing this issue :

checking for libheif >= 1.4.0... no

I have 1.14.2 version installed for libheif

**Comment by dlemstra** [comment]
You can probably find more info on why this failed in the `config.log` file.
